src/directory-sync/interfaces/directory_user.interface.ts

```diff
@@ -24,8 +24,8 @@ export interface User<
   state: 'active' | 'inactive' | 'suspended';
 }
 
-export interface UserWithGroups<
+export interface DirectoryUserWithGroups<
```

Comment:
is there any reason to keep this separate `DirectoryUserWithGroups` interface around? seems like all uses of `DirectoryUser` are actually `DirectoryUserWithGroups`